rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java

```diff
@@ -838,11 +846,19 @@ public void fetchNodeSourceConfiguration(String nodeSourceName, Runnable success
                                       new AsyncCallback<String>() {
 
                                           public void onSuccess(String result) {
-                                              model.setEditedNodeSourceConfiguration(parseNodeSourceConfiguration(result));
-                                              success.run();
+                                              try {
+                                                  model.setEditedNodeSourceConfiguration(nodeSourceConfigurationParser.parseNodeSourceConfiguration(result));
+                                                  success.run();
+                                              } catch (ImportException e) {
+                                                  runFailure(e);
+                                              }
                                           }
 
                                           public void onFailure(Throwable caught) {
+                                              runFailure(caught);
+                                          }
+
+                                          private void runFailure(Throwable caught) {
                                               String msg = JSONUtils.getJsonErrorMessage(caught);
                                               SC.warn("Failed to fetch configuration of node source " + nodeSourceName +
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "pluginName" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)